pkg/trace/api/otlp.go

```diff
@@ -20,13 +20,13 @@ import (
 	"sync"
 	"time"
 
+	pb "github.com/DataDog/datadog-agent/pkg/proto/pbgo/trace"
+	"github.com/DataDog/datadog-agent/pkg/proto/pbgo/trace/otlp"
```

Comment:
It probably should be alised to `otlppb` or `pbgotlp` to avoid confusion with non-protobuf code. The former would let the patch be smaller too.